data-protocols/dsp/dsp-http-spi/src/main/java/org/eclipse/edc/protocol/dsp/http/spi/dispatcher/DspHttpRemoteMessageDispatcher.java

```diff
@@ -41,10 +42,10 @@ <M extends RemoteMessage, R> void registerMessage(Class<M> clazz, DspHttpRequest
     /**
      * Registers a {@link Policy} scope to be evaluated for certain types of messages
      *
-     * @param messageClass the message type for which evaluate the policy.
-     * @param scope the scope to be used.
+     * @param <M>            the message type.
+     * @param messageClass   the message type for which evaluate the policy.
      * @param policyProvider function that extracts the Policy from the message.
-     * @param <M> the message type.
```

Comment:
@paullatzelsperger , in fact it was already there before, I will not remove it

Reply:
so long as `javadoc` doesn't vomit all over itself i'm fine with it :)